doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/system-design-loads-and-air-flow-rates.tex

```diff
@@ -164,26 +154,15 @@ \subsubsection{DuringDay}\label{duringday}
 
 \emph{CoolOutHumRat\(_{sys}\)} = \emph{W\(_{outside}\)}
 
-Here \emph{r\(_{air}\)} is the density of dry air at 20C and standard elevation corrected pressure, {[}kg/m\(^{3}\){]};
-
-\begin{lstlisting}
-        *FracOA* is the outside air fraction;
-\end{lstlisting}
+Here \emph{\rho\(_{air}\)} is the density of dry air at 20\(^{\circ}\)C and standard elevation corrected pressure, {[}kg/m\(^{3}\){]};
 
-\begin{itemize}
-\item
-  C\(_{p,air}\)* is the specific heat of dry air at 20C, {[}J/kg-K{]};
-\item
-  T\(_{sup}\)* is the user specified design cooling supply temperature {[}C{]};
-\item
-  T\(_{mix}\)* is the current mixed air temperature {[}C{]};
-\item
-  W\(_{mix}\)* is the current mixed air humidity ratio {[}kg water / kg dry air{]};
-\item
-  \(T_{outside}\) is the current outside air temperature {[}C{]};
-\item
-  \(W_{outside}\) is the current outside air humidity ratio {[}kg water / kg dry air{]}.
-\end{itemize}
+{\setlength\parindent{25pt} FracOA is the outside air fraction; }
+{\setlength\parindent{25pt} C\(_{p,air}\) is the specific heat of dry air at 20\(^{\circ}\)C, {[}J/kg-K{]}; }
+{\setlength\parindent{25pt} T\(_{sup}\) is the user specified design cooling supply temperature {[}C{]}; }
+{\setlength\parindent{25pt} T\(_{mix}\) is the current mixed air temperature {[}C{]}; }
+{\setlength\parindent{25pt} W\(_{mix}\) is the current mixed air humidity ratio {[}kg water / kg dry air{]}; }
+{\setlength\parindent{25pt} \(T_{outside}\) is the current outside air temperature {[}C{]}; }
+{\setlength\parindent{25pt} \(W_{outside}\) is the current outside air humidity ratio {[}kg water / kg dry air{]}. }
```

Comment:
I'm a little surprised about this.  Is there a reason you didn't leave it as the `itemize` environment?  Seems like it would've been cleaner than forcing the indent in this fashion.  It can stay like this here since it looks like this PR is otherwise good to go, but moving forward, the itemize environment it a good option.

Reply:
@Myoldmopar Was going for consistency.  A few parameter lists showed up as itemize.  Nice but then we have some lists with bullets and some without.  I don't remember why I used the parindent for this.  I'd have to look at it again.